site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
Same story for "Workspace settings" - the form should switch to defaults for ephemeral, unless this is expected to show current values.

<img width="1118" alt="Screenshot 2023-07-18 at 15 41 45" src="https://github.com/coder/coder/assets/14044910/395574a6-9b89-4ec0-ba24-57bd2400298c">

Reply:
What about this @BrunoQuaresma ? It does not seem to be fixed.

Reply:
When I checked it last time, it showed values for the current build instead of default ephemerals. Would you mind giving it a sanity check?

Reply:
Yeap

Reply:
I apologize for the noise, everything is perfect. I must have not rebuilt the project or used the wrong branch.